src/neo4j_genai/schema.py

```diff
@@ -138,3 +145,54 @@ def get_schema(
             "\n".join(formatted_rels),
         ]
     )
+
+
+def get_structured_schema(driver: neo4j.Driver) -> dict[str, Any]:
```

Comment:
Can't this function be used in the `get_schema` function? Seems to be a lot of duplicated code.

Reply:
I couldn't find a way of doing that without making the code unreadable. 🤷‍♀️

Reply:
[1f2561e](https://github.com/neo4j/neo4j-genai-python/pull/43/commits/1f2561efa30cae676a8e2e31b7b09297c85a8bfc)